src/dbnode/storage/index/compaction/plan.go

```diff
@@ -35,19 +35,11 @@ var (
 
 var (
 	// DefaultLevels are the default Level(s) used for compaction options.
-	DefaultLevels = []Level{ // i.e. tiers for compaction [0, 262K), [262K, 1M), [1M, 4M)
-		Level{
+	DefaultLevels = []Level{ // i.e. tiers for compaction [0, 262K)
+		{
 			MinSizeInclusive: 0,
 			MaxSizeExclusive: 1 << 18,
 		},
-		Level{
```

Comment:
Do we not need these levels any longer?

Reply:
In practice I found while testing the new active block semantics that the more extra levels of compactions caused more churn and really huge segments needing multiple GC passes. Whereas with just a fixed ceiling it got a lot better in terms of how difficult the speed of re-processing segments got, since things wouldn't get re-combined any longer at higher levels of compaction.